uo_cli.gemspec

```diff
@@ -33,6 +33,8 @@ Gem::Specification.new do |spec|
 
   #   use specific versions of urbanopt and openstudio dependencies while under heavy development
   spec.add_dependency 'optimist'
+  spec.add_dependency 'os', '~> 1.1.1'
+  spec.add_dependency 'pycall', '1.2.1'
```

Comment:
I don't see `os` being used anywhere. Can we remove it?
Any reason we can't use pycall 1.3.1 instead?